utils/constants.go

```diff
@@ -10,5 +10,6 @@ const (
 
 // API Constants
 const (
-	APIUsers = "user"
+	APIUsers = "/user"
+	APISA    = "/api/v2/stack-analyses"
 )
```

Comment:
I guess I raised this. It doesn't make sense to have a general const file to define endpoints. Rather it should be declared in the respective api.go where the actual http request is made.

Reply:
As you told, don't define everything in one file, just following that paradigm.

Reply:
Could you please point to the discussion.?

Reply:
It happened overcall. I hadn't recorded. :)

Reply:
Okay, but I don't see a point now :) Please move it to the respective file where the http call is made :)

Reply:
Ack. Added constants in respective files